### DISCUSSION TITLE
Welcome to InfusePlus Discussions!

### DISCUSSION THREAD

**Discussion by dayanch96** [dayanch96/InfusePlus#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by acjiao2012** [comment]
how to use？

**Comment by YouuKay** [comment]
I tried with sideloadly but i have to refresh then every 7 days, Please share a direct method to inject to the app is possible. because when i sideload this the app does not load movie posters and thumbnail art
